moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -351,7 +351,7 @@ void RobotStateDisplay::unsetLinkColor(rviz::Robot* robot, const std::string& li
 // ******************************************************************************************
 void RobotStateDisplay::loadRobotModel()
 {
-  load_robot_model_ = false;
+  robot_model_loaded_ = false;
```

Comment:
I'm not sure in which thread `Display::reset()` is processed, but it can call `robot_->clear()` and will call `loadRobotModel()` only *afterwards*.
Shouldn't you set `robot_model_loaded_ = false` *before* clearing the model, instead of in here?

Reply:
@v4hn Thanks for the comment. Added setting `robot_model_loaded_` to `false` also in the top of lines in `reset`.